### ISSUE TITLE
Adding the abandoned mod "ships"

### ISSUE THREAD

**Issue by snouz** [rudegrass/cargo_ships#84]
Animation is a bit choppy though. But its license is open
https://mods.factorio.com/mod/ships_extended
https://mods.factorio.com/mod/SteamShip
https://mods.factorio.com/mod/Ships (original)

**Comment by squidy5** [comment]
Whats the intended use for this? 
Personally I am against adding stuff just for variety if there is no game play mechanic warranting such a change.

Additions I could well imagine are military boats like a patrol boat and a artillery boat, maybe even (if possible to code) a boat acting as a mobile drone port for automatic construction of harbor-infrastrucutre at far away shores -
but I am not sure if those grapics are suited for any of the use cases.

If you have certain content you want to add, I could also imagine to create 3d-models if needed.

**Comment by snouz** [comment]
People were suggesting a steam / early version of the boat, and this mod was completely abandonned since 1.6, so I figured it would be an easy addition, but I totally get it, it's not a good thing to add too much content.
What I could do down the road if I feel motivated for it, is taking the mod back, update it, and make cargo ship a dependency to borrow its logic (and maybe add the compatibility myself).

I was drawn to it because I like it's graphics. But I checked the sprites, and there's actually pretty few original sprite directions (some are just rotated). It might be a good starting boat (no technology, high pollution and fuel) for water based games.

**Comment by squidy5** [comment]
Yeah i was also going to suggest adding it as a sub-mod if you really want it to be in the game :)

**Comment by GeneralTanker** [comment]
A ship like that could be a more medium size ship that carries only 250-500 of storage space by default. Useful for moving smaller amount of items that can't fit on the boat but would be a waste to bring out the bigger cargo ship for.

**Comment by robot256** [comment]
Now that I pushed the Ship API to production, the boats from this mod can be released in a separate mod and register the entities with the API. No changes are needed to Cargo Ships itself, and the boats from the other mod will be able to use Shift-Enter and even waterways with minimal code on their side.

This would satsify the requests for early-game water travel very nicely.  I'll look into if no one beats me to it.

**Comment by robot256** [comment]
It was easier than I expected.  Right now it's just an independent boat.  It will take a little more work to make a prototype that goes on waterways.  https://mods.factorio.com/mod/steamship-reloaded
